### PR TITLE
Simplify opencl make

### PR DESCRIPTION
**Pull request by jean-m-cyr**

### CODE REVIEW

**Comment by jean-m-cyr** [comment]
Travis build failure appears to be unrelated to this PR.

File: libethash-cl/kernels/cl/ethash.cl [hunk 1]
```diff
@@ -1,3 +1,5 @@
+static const char *ethash_opencl_kernel = R"foo(
```
Comment (chfast) [inline libethash-cl/kernels/cl/ethash.cl]: Something nicer than `foo`? :)

Comment (jean-m-cyr) [reply libethash-cl/kernels/cl/ethash.cl]: Agreed.

**Comment by jean-m-cyr** [comment]
Not as much of a simplification as I had hoped. Instead of converting OpenCL source to an array of hexadecimals bytes keep it as an ascii (utf8?) string and wrap it in C++ raw string delimiters.

**Comment by jean-m-cyr** [comment]
@StefanOberhumer As far as I can tell the ethash.h file is not built in the source directory.

**Comment by StefanOberhumer** [comment]
> @StefanOberhumer As far as I can tell the ethash.h file is not built in the source directory.
You're right. Dammed. Viewing and working on too much repos. Not knowing what I've seen where....

**Comment by jean-m-cyr** [comment]
Need to re-purpose this repo soon. Will need to delete this PR.

**Pull request closed without merging**